packages/ethereumjs-config-lint/.eslintrc.js

```diff
@@ -0,0 +1,33 @@
+module.exports = {
+  parser: "@typescript-eslint/parser",
+  plugins: ["@typescript-eslint"],
+  env: {
+    es6: true,
+    node: true
+  },
+  ignorePatterns: ["node_modules/**/*", "**/node_modules/**/*", "dist/**/*"],
```

Comment:
Why is here this double asterisk path for `node_modules` but not for `dist`?

Reply:
The `dist` directories that are under a module's dependencies would be excluded by the first rule already.

Reply:
@evertonfraga thanks, i will update :)